ag/ag.proto

```diff
@@ -65,6 +65,9 @@ message Course {
     repeated Enrollment enrollments = 10;
     repeated Assignment assignments = 12;
     repeated Group groups = 13;
+    // or rename all Path fields to something else? See also scm structs.
+    // Name sounds best, but would be misleading
```

Comment:
Update this comment so it is useful instead of a question. Maybe something like:
```go
// The organization's SCM name, e.g. uis-dat520-2020.
```

Reply:
Fixed.